### ISSUE TITLE
geanynumberedbookmarks doesn't move me between files

### ISSUE THREAD

**Issue by uvtc** [geany/geany-plugins#990]
geanynumberedbookmarks won't navigate me between files. It only works within a single file.

Is it intended that it be able to move the cursor to different files?

Regular Geany bookmarks only work within the current file.

**Comment by elextr** [comment]
I don't think it is meant to work between files, its not explicit, but the [doc](https://plugins.geany.org/geanynumberedbookmarks.html) for the plugin talks about saving bookmarks and other settings "per-file".

You could check the source of the plugin.
